modules/manage/pages/monitor/monitoring-n1ql-query.adoc

```diff
@@ -1269,7 +1269,12 @@ For each completed request, this catalog maintains information such as requestId
 This information provides a general insight into the health and performance of the query engine and the cluster.
 
 For field names and meanings, refer to xref:n1ql:n1ql-rest-api/admin.adoc#_requests[Requests].
-Note that most field names and meanings match exactly those of `system:active_requests`.
+Most field names and meanings match exactly those of `system:active_requests`.
+
+Note that the `completed` state means that the request was started and completed by the Query service, but it does not mean that it was necessarily successful.
+The request could have been successful, or completed with errors.
+
+To find requests that completed successfully, search for completed requests whose `errorCount` field has the value `0`.
```

Comment:
...search for completed requests whose `state` field is "completed" and `errorCount` field has the value 0.